README.md

```diff
@@ -61,7 +69,7 @@ Run the following shell command to build the Docker image from scratch. This wil
 npm run dev
 ```
 
-After the Docker image has finished building, the application can be accessed at [localhost:5000](localhost:5000).
+After the Docker image has finished building, the React application can be accessed at [localhost:3000](localhost:3000).
```

Comment:
May want to also add a note on the port for the server:

```
The server can be accessed at `localhost:5000`.
```

Reply:
done